pcsx2/GS/Renderers/HW/GSRendererHW.cpp

```diff
@@ -2054,6 +2054,13 @@ void GSRendererHW::Draw()
 						ds_end_bp, m_cached_ctx.ZBUF.PSM);
 				}
 
+				GSUploadQueue clear_queue;
```

Comment:
What's happening here? Wouldn't we want to remove contained transfers from the queue, not add more to it?

Reply:
The problem is, this was happening and killing the target, then doing a draw with the alpha masked, so there was no way to know the alpha was valid.  What I'm thinking however, is adding a variable to the GSUploadQueue so it doesn't actually upload anything, just marks that there was a clear in that format.

Reply:
Okay I did some modification so it only stores it on zero clears and it uses it to skip preloading the RT rather than the usual use of the transfer list